samples/ingestion/ingestion-client/FetchTranscription/Program.cs

```diff
@@ -44,6 +46,20 @@ public static void Main(string[] args)
                     s.AddSingleton(blobServiceClient);
                     s.AddSingleton(storageCredential);
                     s.AddTransient<IStorageConnector, StorageConnector>();
+
+                    s.AddAzureClients(clientBuilder =>
```

Comment:
Is there any reason for using AddAzureClients over AddSingleton?

Reply:
no, I think both should work and do the same thing:
https://stackoverflow.com/questions/77613026/is-using-addazureclients-and-a-clientfactory-for-azure-servicebus-di-preferred-o